api_specs/v2/component_analyses.yaml

```diff
@@ -120,74 +272,84 @@ components:
             properties:
               vendor_cve_ids:
                 type: string
+                example: "CVE-1234"
               cvss:
                 type: string
+                example: 9.2
               is_private:
                 type: boolean
+                example: true
 
     RegisteredUser:
-      title: Vendor Private Recommendation Information
-      description: Vendor Recommendation information for Premium Response.
+      title: Vendor Private Recommendation
       type: object
       properties:
         recommendation:
           type: object
           properties:
             recommended_version:
               type: string
-            affected_version:
-              type: string
+              example: 2
             component_analyses:
               $ref: '#/components/schemas/VulnerabilityDetails'
             message:
               type: string
+              example: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
             severity:
               type: string
+              example: "high"
             public_vulnerabilities_count:
               type: integer
+              example: 0
             private_vulnerabilities_count:
               type: integer
+              example: 0
             exploitable_vulnerabilities_count:
               type: integer
+              example: 0
             vendor_package_link:
               type: string
+              example : "https://snyk.io/vuln/maven:org.webjars%3Ajquery-form"
 
     FreeUser:
-      title: Vendor Public Recommendation Information
-      description: Vendor Recommendation information for Standard Response.
+      title: Vendor Public Recommendation
       type: object
       properties:
         recommendation:
           type: object
           properties:
             recommended_version:
               type: string
-            affected_version:
-              type: string
+              example: "3.1"
             component_analyses:
               $ref: '#/components/schemas/VulnerabilityDetails'
             message:
               type: string
+              example: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
             severity:
```

Comment:
This field should be called as `highest_severity`. And severity is actually associated with each item of `VulnerabilityDetails` schema.

Reply:
Ack!